transformer_engine/jax/csrc/modules.h

```diff
@@ -56,28 +63,42 @@ struct CustomCallGemmDescriptor {
     size_t m;
     size_t n;
     size_t k;
+    size_t workspace;
```

Comment:
I prefer to name as workspace_size or wkspace_size for the better readability. `workspace` has been used as a ptr name in some of place.